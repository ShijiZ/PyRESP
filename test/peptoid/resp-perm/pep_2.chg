All values are reported in atomic units
%FLAG TITLE
 molecule 1: peptoid

%FLAG ATOM CRD: I4,3E16.7
 atm.no      X               Y               Z
   1   6.6911500E+00  -2.4155300E+00   1.9019460E+00
   2   5.8593940E+00  -6.3711010E-01   1.3550270E+00
   3   7.1693530E+00   3.8731120E-01   1.6170360E-01
   4   5.4836410E+00   4.7919210E-01   3.0292610E+00
   5   3.4325740E+00  -1.2236530E+00  -4.2293720E-02
   6   2.7091160E+00  -3.3915580E+00  -3.5405420E-01
   7   2.1000450E+00   7.8034240E-01  -9.8437610E-01
   8   2.5933670E+00   3.4257410E+00  -3.3520140E-01
   9   4.5896080E+00   3.7505730E+00  -8.2132520E-02
  10   1.9628530E+00   4.6067420E+00  -1.8827110E+00
  11   1.5824870E+00   3.9493260E+00   1.3602580E+00
  12  -3.6201050E-01   2.1964500E-01  -2.0371270E+00
  13  -8.2865300E-01   1.5992500E+00  -3.4771030E+00
  14  -3.0660460E-01  -1.6413580E+00  -2.8683080E+00
  15  -2.4034790E+00   2.7818140E-01   1.9197500E-03
  16  -1.9891920E+00   9.8704980E-01   2.1548800E+00
  17  -4.7025800E+00  -4.7466670E-01  -8.1969090E-01
  18  -4.8996580E+00  -1.1204850E+00  -2.5775100E+00
  19  -6.9003170E+00  -5.3917450E-01   8.5522230E-01
  20  -6.3087580E+00   1.9254090E-01   2.6622220E+00
  21  -8.4140140E+00   6.2202240E-01   1.1308310E-01
  22  -7.5802200E+00  -2.4550250E+00   1.0958760E+00

%FLAG ATOM CHRG: 2(I4,X7),I4,X2,E16.7
 atm.no   element.no   ivary      q(opt)
   1          1         -1    -5.0428915E-03
   2          6         -1     1.4469516E-02
   3          1         -1     3.9507347E-02
   4          1         -1     7.9773989E-02
   5          6         -1     2.8836796E-01
   6          8         -1    -4.1707592E-01
   7          7         -1    -2.6137743E-01
   8          6          0    -5.4202229E-03
   9          1          0     4.3732579E-02
  10          1          9     4.3732579E-02
  11          1          9     4.3732579E-02
  12          6          0    -3.0689095E-03
  13          1          0     1.6542661E-01
  14          1         13     1.6542661E-01
  15          6         -1     1.3836267E-01
  16          8         -1    -3.3054706E-01
  17          7         -1    -1.1117301E-01
  18          1         -1     3.0625535E-02
  19          6         -1    -1.3812735E-02
  20          1         -1     8.8450517E-02
  21          1         -1     1.2177285E-03
  22          1         -1     4.6919664E-03

%FLAG PERM DIP LOCAL: 3(I4,X5),I4,X2,E16.7
 dip.no   atm.no   ref.no   ivary      p(opt)
   1        1        2       -1    -7.1540645E-02
   2        2        1       -1    -5.6902310E-02
   3        2        3       -1     3.6146736E-02
   4        2        4       -1    -1.6607762E-02
   5        2        5       -1     3.1198110E-02
   6        3        2       -1    -5.4521684E-02
   7        4        2       -1    -2.2525593E-02
   8        5        2       -1     5.0748537E-02
   9        5        6       -1    -2.6318891E-01
  10        5        7       -1     6.4297779E-02
  11        6        5       -1     3.2730754E-01
  12        7        5       -1    -1.9965964E-02
  13        7        8       -1     2.6228745E-02
  14        7       12       -1     1.4175465E-02
  15        8        7        0    -5.5423630E-02
  16        8        9        0     1.8453198E-02
  17        8       10       16     1.8453198E-02
  18        8       11       16     1.8453198E-02
  19        9        8        0    -6.2389572E-02
  20       10        8       19    -6.2389572E-02
  21       11        8       19    -6.2389572E-02
  22       12        7        0    -5.6703536E-02
  23       12       13        0    -2.1736144E-02
  24       12       14       23    -2.1736144E-02
  25       12       15        0     2.1228615E-01
  26       13       12        0     8.7238910E-03
  27       14       12       26     8.7238910E-03
  28       15       12       -1     4.9317444E-02
  29       15       16       -1    -1.7247008E-01
  30       15       17       -1     6.1156669E-02
  31       16       15       -1     3.4831946E-01
  32       17       15       -1    -1.2478737E-01
  33       17       18       -1     4.6028233E-02
  34       17       19       -1     4.4774118E-02
  35       18       17       -1    -3.6379486E-01
  36       19       17       -1    -9.0966339E-02
  37       19       20       -1    -8.2389414E-02
  38       19       21       -1     1.0511028E-01
  39       19       22       -1     6.6966367E-02
  40       20       19       -1    -2.2507811E-02
  41       21       19       -1    -4.9318733E-02
  42       22       19       -1    -5.8166513E-02

%FLAG PERM DIP GLOBAL: I4,3E16.7
 atm.no      X               Y               Z
   1   2.9196474E-02  -6.2426469E-02   1.9198066E-02
   2  -2.3502508E-02   5.2291924E-02  -6.5163695E-02
   3   3.4893806E-02   2.7287845E-02  -3.1786945E-02
   4  -4.1347758E-03   1.2283759E-02   1.8423225E-02
   5   9.2441526E-02   3.0762496E-01   3.6919196E-02
   6   1.0265906E-01   3.0762682E-01   4.4238970E-02
   7  -1.8377542E-02   3.7641902E-02  -6.5820423E-03
   8   1.3129510E-02   7.1287250E-02   1.6653039E-02
   9   6.1103153E-02   9.9428172E-03   7.7462122E-03
  10  -1.9224286E-02   3.6008560E-02  -4.7183357E-02
  11  -3.0882484E-02   1.5995574E-02   5.1796452E-02
  12  -1.9683584E-01  -2.1180647E-03   1.5231212E-01
  13   1.9876936E-03  -5.8765159E-03   6.1336700E-03
  14  -2.3706228E-04   7.9625745E-03   3.5563299E-03
  15  -5.1156081E-02  -7.2079883E-02  -2.1566077E-01
  16  -6.2626438E-02  -1.0715736E-01  -3.2545610E-01
  17  -1.5271088E-01  -5.3601453E-02  -5.5963961E-02
  18  -3.8074533E-02  -1.2476903E-01  -3.3960230E-01
  19  -1.9621901E-01  -3.4762323E-02  -4.8186375E-02
  20   6.5354579E-03   8.0838854E-03   1.9963470E-02
  21  -3.6468819E-02   2.7976193E-02  -1.7880025E-02
  22  -1.9318726E-02  -5.4436871E-02   6.8379210E-03

%FLAG IND DIP GLOBAL: I4,3E16.7
 atm.no      X               Y               Z
   1  -4.8812381E-03  -2.9872708E-02  -1.6352454E-03
   2  -7.7589677E-02  -1.2473739E-01  -4.3237023E-02
   3   1.1819930E-02  -1.4184030E-02   1.5649481E-03
   4   5.0125409E-03  -1.5080254E-02   7.0691731E-03
   5  -1.9790955E-01  -1.6145020E-01  -9.5819686E-02
   6   2.9453534E-03  -9.7248878E-02   1.8009912E-02
   7  -3.5551945E-02   6.6082855E-02   2.7969742E-03
   8  -5.1245888E-02  -7.6359433E-02   3.5210839E-02
   9  -1.3946145E-02  -4.3560251E-03  -3.6640041E-03
  10   5.5873010E-03  -9.9954021E-03   8.7323556E-03
  11  -2.2688981E-02  -2.0258559E-02   1.1711002E-02
  12   5.0756254E-02  -4.4487398E-02   2.6080602E-01
  13   1.5485634E-02   2.2627422E-03   9.6762684E-03
  14   4.2983275E-02  -2.8989366E-02   2.5813477E-02
  15  -1.2594689E-01  -4.7101801E-04   2.5547720E-01
  16   8.3615589E-03  -7.6660053E-03   3.3682176E-02
  17  -1.0930983E-01  -1.3413159E-02   5.9633745E-02
  18   1.2229023E-02   4.0799112E-04   1.2089578E-02
  19   5.0264704E-02   1.0071388E-02   1.4774919E-02
  20   3.0023092E-02   3.8958664E-03   1.0591561E-03
  21   3.5585465E-03   3.7951849E-03   3.8758550E-03
  22   7.1295670E-03  -4.6395759E-03   6.3565155E-03

