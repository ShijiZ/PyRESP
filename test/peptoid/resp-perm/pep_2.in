 second stage of RESP. Peptoid for resp charge calculations
 &cntrl 
 nmol = 1,
 iqopt = 2,
 ihfree = 1,
 qwt = 0.001,
 ioutopt = 1,
 ipol = 5,
 igdm = 1,
 exc12 = 0,
 exc13 = 0,
 ipermdip = 1,
 pwt = 0.001,
 virtual = 0 
 &end 
    1.0
peptoid
    0   22 42
    1   -1 -1
    6   -1 -1 -1 -1 -1
    1   -1 -1
    1   -1 -1
    6   -1 -1 -1 -1
    8   -1 -1
    7   -1 -1 -1 -1
    6    0  0  0  16 16
    1    0  0
    1    9 19
    1    9 19
    6    0  0  0  23 0
    1    0  0
    1   13 26
    6   -1 -1 -1 -1
    8   -1 -1
    7   -1 -1 -1 -1
    1   -1 -1
    6   -1 -1 -1 -1 -1
    1   -1 -1
    1   -1 -1
    1   -1 -1




