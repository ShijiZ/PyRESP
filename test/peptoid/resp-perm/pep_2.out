
 -----------------------------------------------
              Py_RESP Version 1.0
 -----------------------------------------------
  second stage of RESP. Peptoid for resp charge calculations
 -----------------------------------------------

 ------------------
 Control Parameters
 ------------------
 nmol        = 1   iqopt       = 2
 ihfree      = 1   irstrnt     = 1
 ioutopt     = 1   ireornt     = 0
 iquad       = 1   qwt         = 0.00100000
 ipol        = 5   igdm        = 1
 exc12       = 0   exc13       = 0
 ipermdip    = 1   pwt         = 0.00100000
 virtual     = 0

 --------------------
 Molecule Information
 --------------------
 Single-molecule run

 Molecule 1 weight:     1.000 
 Molecule 1 name: peptoid
 Total charge (ich):  0
 Number of centers: 22
 Number of permanent dipoles: 42
    1    1   -1   -1
    2    6   -1   -1   -1   -1   -1
    3    1   -1   -1
    4    1   -1   -1
    5    6   -1   -1   -1   -1
    6    8   -1   -1
    7    7   -1   -1   -1   -1
    8    6    0    0    0   16   16
    9    1    0    0
   10    1    9   19
   11    1    9   19
   12    6    0    0    0   23    0
   13    1    0    0
   14    1   13   26
   15    6   -1   -1   -1   -1
   16    8   -1   -1
   17    7   -1   -1   -1   -1
   18    1   -1   -1
   19    6   -1   -1   -1   -1   -1
   20    1   -1   -1
   21    1   -1   -1
   22    1   -1   -1

 -----------------------------
 Atom/Dipole Ivary Information
 -----------------------------
    1   -1   -1
    6   -1   -1   -1   -1   -1
    1   -1   -1
    1   -1   -1
    6   -1   -1   -1   -1
    8   -1   -1
    7   -1   -1   -1   -1
    6    0    0    0   16   16
    1    0    0
    1    9   19
    1    9   19
    6    0    0    0   23    0
    1    0    0
    1   13   26
    6   -1   -1   -1   -1
    8   -1   -1
    7   -1   -1   -1   -1
    1   -1   -1
    6   -1   -1   -1   -1   -1
    1   -1   -1
    1   -1   -1
    1   -1   -1

 -------------------------
 Input Information Summary
 -------------------------
 Total number of atoms      =   22
 Total number of permanent dipoles      =   42
 Weight factor on initial charge restraints=  0.001000
 Weight factor on initial dipole restraints=  0.001000
 There are  1 charge constraints

 Read polarizability and radii information from:
 ../../../polarizability/pGM-pol-2016-09-01
 Assume the units are in a.u.

 --------------------------------------
 Atomic and ESP Coordinates Information
 --------------------------------------
 Reading esp's for molecule   1
 total number of atoms      =    22
 total number of esp points =   983

 center       X               Y               Z
    1   6.6911500E+00  -2.4155300E+00   1.9019460E+00
    2   5.8593940E+00  -6.3711010E-01   1.3550270E+00
    3   7.1693530E+00   3.8731120E-01   1.6170360E-01
    4   5.4836410E+00   4.7919210E-01   3.0292610E+00
    5   3.4325740E+00  -1.2236530E+00  -4.2293720E-02
    6   2.7091160E+00  -3.3915580E+00  -3.5405420E-01
    7   2.1000450E+00   7.8034240E-01  -9.8437610E-01
    8   2.5933670E+00   3.4257410E+00  -3.3520140E-01
    9   4.5896080E+00   3.7505730E+00  -8.2132520E-02
   10   1.9628530E+00   4.6067420E+00  -1.8827110E+00
   11   1.5824870E+00   3.9493260E+00   1.3602580E+00
   12  -3.6201050E-01   2.1964500E-01  -2.0371270E+00
   13  -8.2865300E-01   1.5992500E+00  -3.4771030E+00
   14  -3.0660460E-01  -1.6413580E+00  -2.8683080E+00
   15  -2.4034790E+00   2.7818140E-01   1.9197500E-03
   16  -1.9891920E+00   9.8704980E-01   2.1548800E+00
   17  -4.7025800E+00  -4.7466670E-01  -8.1969090E-01
   18  -4.8996580E+00  -1.1204850E+00  -2.5775100E+00
   19  -6.9003170E+00  -5.3917450E-01   8.5522230E-01
   20  -6.3087580E+00   1.9254090E-01   2.6622220E+00
   21  -8.4140140E+00   6.2202240E-01   1.1308310E-01
   22  -7.5802200E+00  -2.4550250E+00   1.0958760E+00


 ------------------------
 Optimization Information
 ------------------------
 Since iqopt>1,   22 new q0 values and   42 new p0 values will be
 read from file pep_1.chg

 Number of unique UNfrozen charge centers =     4
 Number of unique UNfrozen permanent dipoles =     7

 Non-linear optimization requested.
  qchnge =    3.3898617104E-02
  qchnge =    2.1273469737E-03
  qchnge =    3.4284139733E-05
  qchnge =    4.1232517225E-06
  qchnge =    8.5366621135E-07

 Convergence in    4 iterations

 -----------------------
 Fitting Results Summary
 -----------------------
 molecule 1: peptoid

          Point Charges Before & After Optimization
 atm.no   element.no   q(init)        q(opt)     ivary    d(rstr)/dq
    1         1       -0.005043     -0.005043     -1       0.000000
    2         6        0.014470      0.014470     -1       0.009897
    3         1        0.039507      0.039507     -1       0.000000
    4         1        0.079774      0.079774     -1       0.000000
    5         6        0.288368      0.288368     -1       0.003276
    6         8       -0.417076     -0.417076     -1       0.002332
    7         7       -0.261377     -0.261377     -1       0.003573
    8         6       -0.008855     -0.005420      0       0.009985
    9         1        0.056022      0.043733      0       0.000000
   10         1        0.040183      0.043733      9       0.000000
   11         1        0.032329      0.043733      9       0.000000
   12         6       -0.001046     -0.003069      0       0.009995
   13         1        0.189461      0.165427      0       0.000000
   14         1        0.145467      0.165427     13       0.000000
   15         6        0.138363      0.138363     -1       0.005858
   16         8       -0.330547     -0.330547     -1       0.002896
   17         7       -0.111173     -0.111173     -1       0.006688
   18         1        0.030626      0.030626     -1       0.000000
   19         6       -0.013813     -0.013813     -1       0.009906
   20         1        0.088451      0.088451     -1       0.000000
   21         1        0.001218      0.001218     -1       0.000000
   22         1        0.004692      0.004692     -1       0.000000

 Sum over the calculated charges:     -0.000

       Permanent Dipoles Before & After Optimization
 dip.no   element.no   p(init)        p(opt)     ivary    d(rstr)/dp
    1         1       -0.071541     -0.071541     -1       0.000000
    2         6       -0.056902     -0.056902     -1       0.008691
    3         6        0.036147      0.036147     -1       0.009404
    4         6       -0.016608     -0.016608     -1       0.009865
    5         6        0.031198      0.031198     -1       0.009546
    6         1       -0.054522     -0.054522     -1       0.000000
    7         1       -0.022526     -0.022526     -1       0.000000
    8         6        0.050749      0.050749     -1       0.008917
    9         6       -0.263189     -0.263189     -1       0.003552
   10         6        0.064298      0.064298     -1       0.008411
   11         8        0.327308      0.327308     -1       0.002922
   12         7       -0.019966     -0.019966     -1       0.009806
   13         7        0.026229      0.026229     -1       0.009673
   14         7        0.014175      0.014175     -1       0.009901
   15         6       -0.064212     -0.055424      0       0.008748
   16         6       -0.006398      0.018453      0       0.009834
   17         6        0.020207      0.018453     16       0.009834
   18         6        0.046071      0.018453     16       0.009834
   19         1       -0.068622     -0.062390      0       0.000000
   20         1       -0.060608     -0.062390     19       0.000000
   21         1       -0.035680     -0.062390     19       0.000000
   22         6       -0.043922     -0.056704      0       0.008703
   23         6       -0.004606     -0.021736      0       0.009773
   24         6       -0.053349     -0.021736     23       0.009773
   25         6        0.220596      0.212286      0       0.004259
   26         1        0.033464      0.008724      0       0.000000
   27         1       -0.010401      0.008724     26       0.000000
   28         6        0.049317      0.049317     -1       0.008969
   29         6       -0.172470     -0.172470     -1       0.005016
   30         6        0.061157      0.061157     -1       0.008531
   31         8        0.348319      0.348319     -1       0.002759
   32         7       -0.124787     -0.124787     -1       0.006253
   33         7        0.046028      0.046028     -1       0.009084
   34         7        0.044774      0.044774     -1       0.009127
   35         1       -0.363795     -0.363795     -1       0.000000
   36         6       -0.090966     -0.090966     -1       0.007397
   37         6       -0.082389     -0.082389     -1       0.007718
   38         6        0.105110      0.105110     -1       0.006893
   39         6        0.066966      0.066966     -1       0.008309
   40         1       -0.022508     -0.022508     -1       0.000000
   41         1       -0.049319     -0.049319     -1       0.000000
   42         1       -0.058167     -0.058167     -1       0.000000

 --------------------------
 Fitting Statistics Summary
 --------------------------
 The initial sum of squares  (ssvpot = sum_i(espi**2))          4.4905295E-01
 The residual sum of squares (RSS = sum_i((espi-espclci)**2))   1.7701911E-03
 The root-mean-squared error (RMSE = sqrt(RSS/N))               1.3419407E-03
 The relative RMSE           (RRMSE = sqrt(RSS/ssvpot))         6.2785780E-02

 ---------------------------
 Molecular Multipole Summary
 ---------------------------
 Dipole Moments (Debye):
 #MOL         D          Dx         Dy         Dz
   1        3.86374   -1.20417    2.56383   -2.62778

 Traceless Quadrupole Moments (Debye*Angst.):
 #MOL          X          Y          Z
   1    X   10.62254
        Y    7.77926   -8.07087
        Z    6.03206   -1.40993   -2.55167

 Traceless Quadrupole Moments in Principal Axes (Debye*Angst.):
 #MOL          X          Y          Z
   1    X   15.01121
        Y    0.00000   -2.80109
        Z    0.00000    0.00000  -12.21012